chapters/synchronous.tex

```diff
@@ -1096,43 +1096,46 @@ \section{Clocked Discretized Partition}\label{continuous-time-equations-in-clock
 have only a value at a clock tick. Therefore, such a partition is
 handled in the same way as any other clocked partition. Especially,
 operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate
-signals of the discretized continuous-time partition with other
-partitions. Hereby, a discretized continuous-time partition is seen as a
+signals of the discretized partition with other
+partitions. Hereby, a discretized partition is treated similarly as a
 clocked partition.
 
 
-\subsection{Clocked Discrete-Time and Discretized Continuous-Time}\label{clocked-discrete-time-and-clocked-discretized-continuous-time-partition}
+\subsection{Clocked and Discretized Partition}\label{clocked-discrete-time-and-clocked-discretized-continuous-time-partition}\label{clocked-and-discretized-partition}
```

Comment:
Wouldn't it read nicer in the plural, to make it not sound as something which is both at the same time?
```suggestion
\subsection{Clocked and Discretized Partitions}\label{clocked-discrete-time-and-clocked-discretized-continuous-time-partition}\label{clocked-and-discretized-partition}
```